Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocMultilineTextValue.css

```diff
@@ -1,15 +1,27 @@
 /* Style applied to the BocMultilineTextValue. */
 
+:root
+{
+  --min-height: 4.5em;
```

Comment:
Each control should be isolated. Refining root-properties such as this would bleed into other css files, correct?